java/javax/websocket/OnIdleSession.java

```diff
@@ -0,0 +1,12 @@
+package javax.websocket;
```

Comment:
I'm afraid this is not part of WebSocket specification: https://www.jcp.org/en/jsr/detail?id=356 and https://download.oracle.com/otndocs/jcp/websocket-1_1-mrel-eval-spec/index.html
So you cannot add it to `javax.websocket` package

Reply:
Would it be okay if I move it to the package org.apache.tomcat.websocket?
And what about the other chagnes in the javax package?